Framework/API/src/MDGeometry.cpp

```diff
@@ -102,6 +102,50 @@ MDGeometry::MDGeometry(const MDGeometry &other)
     this->setOriginalWorkspace(other.m_originalWorkspaces[i], i);
 }
 
+MDGeometry &MDGeometry::operator=(const MDGeometry &other) {
```

Comment:
Is this duplicated code from the copy constructor? If we can't use copy-and-swap then can we have a common method to do this?